test/integration/generated/bodyComplex/src/models/parameters.ts

```diff
@@ -9,13 +9,14 @@
 import * as coreHttp from "@azure/core-http";
 import * as Mappers from "../models/mappers";
 
-export const $host: coreHttp.OperationParameter = {
+export const $host: coreHttp.OperationQueryParameter = {
```

Comment:
It does seem odd to see something like "$host" as a query parameter :)

Reply:
It is now a Url parameter :smile: thanks for catching this